src/Core/Utilities/CoreHelpers.cs

```diff
@@ -182,26 +184,17 @@ public static Uri GetUri(string uriString)
 
         public static Dictionary<string, string> GetQueryParams(string urlString)
         {
-            var dict = new Dictionary<string, string>();
-            if (!Uri.TryCreate(urlString, UriKind.Absolute, out var uri) || string.IsNullOrWhiteSpace(uri.Query))
+            try
             {
-                return dict;
+                var queryStringParams = urlString.Split('?').Last();
+                var queryStringNameValueCollection = HttpUtility.ParseQueryString(queryStringParams);
```

Comment:
Sorry I missed the above uri creation, but we could even simplify this to use 
```c#
var queryStringNameValueCollection = HttpUtility.ParseQueryString(uri.Query);
```
I made a few tests and it seems to work, keep in mind that `uri.Query` includes a "?" at beginning.